cypress/integration/WorkflowsSearch.js

```diff
@@ -7,23 +7,24 @@ describe('Dockstore workflow search page', function() {
    describe('Select a workflow', function() {
     it('Should display the correct url', function() {
       cy.get('tbody')
-      .children('tr')
+      .children('tr:nth-child(2)')
```

Comment:
the 2nd child seems fragile, is there a way to use an id or something stable?

Reply:
the row are generated without id.  generating a unique id per row seems hacky

Reply:
Maybe a comment on what the 2nd child should be, in case more children are added/removed in the future.

Reply:
The lines below it make it obvious what it's supposed to select